### PR TITLE
Add NeMo Retriever SDG

### PR DESCRIPTION
**Pull request by suhara**
NeMo Retriever Synthetic Data Generation

### CODE REVIEW

File: nemo/retriever-synthetic-data-generation/outputs/.gitkeep
Comment (shubhadeepd) [inline nemo/retriever-synthetic-data-generation/outputs/.gitkeep:1]: Is this mandatory to keep?

Comment (suhara) [reply nemo/retriever-synthetic-data-generation/outputs/.gitkeep:1]: Yes. It's the default output directory.

**Review by shubhadeepd — APPROVED** [review]

File: nemo/retriever-synthetic-data-generation/outputs/.gitkeep
Comment (shubhadeepd) [reply nemo/retriever-synthetic-data-generation/outputs/.gitkeep:1]: Thanks. Changes LGTM!

**Review by dglogo — APPROVED** [review]
lgtm thanks!

**Pull request merged**